Source/GraduatedCylinder.Geo Specs/Devices/Gps/Nmea/GPRMCSpec.cs

```diff
@@ -25,5 +25,18 @@ public class GPRMCSpec
             provideGeoPosition.CurrentLocation.Longitude.Value.ShouldEqualWithinPrecision(longitude, 10);
             provideGeoPosition.CurrentLocation.Longitude.Hemisphere.ShouldEqual(longHemisphere);
         }
+
+        [Theory]
+        [InlineData("$GPRMC,145710.00,A,3605.08678,N,07958.72572,W,0.037,,180716,,,D*6B")]
+        public void EmptyHeading(string nmea) {
```

Comment:
Here lets make this test 'ValidateHeading' and pass in 2 parameters, the sentence and the expected heading. Lets test for both specified and missing headings.